src/cf-forms/src/cf-forms.less

```diff
@@ -10,36 +10,37 @@
 
 // Input borders
 @input-border:                  #5b616b;
-@input-border__hover:           @pacific;
-@input-border__focused:         @pacific;
-@input-border__active:          @pacific;
-@input-border__error:           @red;
-@input-border__warning:         @gold;
-@input-border__success:         @green;
+@input-border__hover:           #0072ce; // @pacific;
```

Comment:
It won't build otherwise. I can add them as a separate (third PR), since any PR off of v4 branch will run into this.